konduit-serving-test/pom.xml

```diff
@@ -149,6 +147,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>com.github.stefanbirkner</groupId>
+            <artifactId>system-rules</artifactId>
+            <version>1.19.0</version>
+        </dependency>
```

Comment:
Is this used anywhere? (I can't see any imports/changes referencing this)
If it is used: it should be test scope, I think, based on the description?
If it's not used: remove it

Reply:
Ah yes this is used with this import: https://github.com/KonduitAI/konduit-serving/blob/6c6e709e67fc6c3f64395d2b4eb77d63c8199cf2/konduit-serving-test/src/test/java/ai/konduit/serving/util/PortsTest.java#L44.

I've added the test scope